bundles/binding/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/command/SatelCommandBase.java

```diff
@@ -1,181 +1,197 @@
-            return false;
-        }
-
-        byte responseCode = response.getPayload()[0];
-        String errorMsg;
-
-        switch (responseCode) {
-            case 0:
-                // success
-                return true;
-            case 0x01:
-                errorMsg = "Requesting user code not found";
-                break;
-            case 0x02:
-                errorMsg = "No access";
-                break;
-            case 0x03:
-                errorMsg = "Selected user does not exist";
-                break;
-            case 0x04:
-                errorMsg = "Selected user already exists";
-                break;
-            case 0x05:
-                errorMsg = "Wrong code or code already exists";
-                break;
-            case 0x06:
-                errorMsg = "Telephone code already exists";
-                break;
-            case 0x07:
-                errorMsg = "Changed code is the same";
-                break;
-            case 0x08:
-                errorMsg = "Other error";
-                break;
-            case 0x11:
-                errorMsg = "Can not arm, but can use force arm";
-                break;
-            case 0x12:
-                errorMsg = "Can not arm";
-                break;
-            case (byte) 0xff:
-                logger.trace("Command accepted");
-                return true;
-            default:
-                if (responseCode >= 0x80 && responseCode <= 0x8f) {
-                    errorMsg = String.format("Other error: %02X", responseCode);
-                } else {
-                    errorMsg = String.format("Unknown response code: %02X", responseCode);
-                }
-        }
-
-        logger.error(errorMsg);
-        return false;
-    }
-
-    protected static int bcdToInt(byte[] bytes, int offset, int size) {
-        int result = 0, digit;
-        int byteIdx = offset;
-        int digits = size * 2;
-        for (int i = 0; i < digits; ++i) {
-            if (i % 2 == 0) {
-                digit = (bytes[byteIdx] >> 4) & 0x0f;
-            } else {
-                digit = (bytes[byteIdx]) & 0x0f;
-                byteIdx += 1;
-            }
-            result = result * 10 + digit;
-        }
-        return result;
-    }
-}
+/**
+ * Copyright (c) 2010-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.satel.command;
+
+import org.openhab.binding.satel.internal.event.EventDispatcher;
+import org.openhab.binding.satel.internal.protocol.SatelMessage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Base class for all command classes.
+ *
+ * @author Krzysztof Goworek
+ * @since 1.9.0
```

Comment:
I think if the file existed since 1.7.0, even if its contents are different, then it makes sense to leave it as it was.  But we're definitely talking about very small stuff!